### ISSUE TITLE
Can you upload *.pb file 

### ISSUE THREAD

**Issue by tingxingdong** [TropComplique/shufflenet-v2-tensorflow#5]
Thank you for sharing pretrained models here: https://drive.google.com/drive/folders/1KGIdE8SmR-Af9zheuQ68EhD0ck4h4riH

I only see the *ckpt files. According to TF, the model also has the graph *pb file. Can you upload the graph?   I know you have a File architecture.py to definition of the graph.
But many users just want the *pb instead of the python source code. 

Thank you.

**Comment by naufalboys** [comment]
bump
